utils/apps.ts

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {AppBinding} from 'mattermost-redux/types/apps';
+import {GlobalState} from 'mattermost-redux/types/store';
+
+export function appsEnabled(state: GlobalState) {// eslint-disable-line @typescript-eslint/no-unused-vars
+    // TODO uncomment when featur flag is in place
+    //return getConfig(state)?.['FeatureFlagApps' as keyof Partial<ClientConfig>];
+    return true;
+}
+
+export function fillBindingsInformation(binding: AppBinding) {
```

Comment:
should we validate the `binding` param?

Reply:
That should never happen, but better safe than sorry.